scripts/build-windows.ps1

```diff
@@ -55,7 +120,7 @@ if ( $env:CLEAN -eq "true" )
 }
 
 # Ensure that the system dependencies are at the correct version
-npm i -gf npm@8.7.0
+npm i -gf npm@$EXPECTED_NPM_VERSION
 npm i -gf pnpm@6.32.8
```

Comment:
With the same code you added above (I don't know powershell), we can get the PNPM version aswell
```suggestion
npm i -gf pnpm@$((Get-Content recipes\package.json | ConvertFrom-Json).engines.pnpm)
```

Reply:
Yes we can, but the package pnpm is not a dependency available on package.json (in the discord we were discussing the possibility of having it or not has such). There is currently an PR open on that I think, to switch from npm to pnpm on the entire project but is a work in progess (https://github.com/ferdium/ferdium-app/pull/26).

Reply:
I should have underlined that: this line is reading from [**recipes**/package.json](https://github.com/ferdium/ferdium-recipes/blob/master/package.json#L7), which does use `pnpm` (and that is why we use `pnpm` in the script)

Reply:
that sounds correct @nathanaelhoun

Reply:
incorporated in the latest commit. tx @nathanaelhoun

Reply:
Wow! That way is fantastic awesome!